cmd/azure-keyvault-env/environment.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 const (
-	envLookupRegex = `^([a-zA-Z0-9]([a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)@azurekeyvault(\?([a-zA-Z_][a-zA-Z0-9_\.]*)?)?$`
+	envLookupRegex = `^([a-zA-Z0-9]([a-zA-Z0-9-]{0,239}[a-zA-Z0-9])?)@azurekeyvault(\?([a-zA-Z_][a-zA-Z0-9_\.]*)?)?$`
```

Comment:
Why 239? Resource names can be up to 253 characters, and the `@azurekeyvault` isn't part of the resource's name in Kubernetes; it's only used in the environment variable's value, so there's no reason not to make it 253 as I did in #636.

Reply:
@tspearconquest , you are correct, I'm good with 253, if the fix in your PR is going to be merged, I can remove this change from here

Reply:
@tspearconquest I have updated it anyways in this PR as well